modules/core/kml/src/Normalizer/KmlNormalizer.php

```diff
@@ -119,6 +119,11 @@ public function denormalize($data, $type, $format = NULL, array $context = []) {
       // Load KML into a Geometry object.
       $geometry = $this->geoPHP->load($placemark['xml'], 'kml');
 
+      // Create an empty collection if no geometry was loaded.
+      if (empty($geometry)) {
+        $geometry = new \GeometryCollection();
```

Comment:
I wonder if it makes sense to add `use GeometryCollection;` to the top of the file and change this to:

`$geometry = new \GeometryCollection();`

Same effect, but feels like it would be more consistent to add a `use` statement in this case, even though `phayes/GeometryCollection` doesn't use namespaces.

I wonder if there's any benefit to consistency in this regard, like IDE integration, code sniffing, etc...

Curious your thoughts on this (very minor) thought @paul121 - doesn't need to be a blocker for merging this, I don't think, but worth considering?

Reply:
Oh interesting, yeah I don't see any problem with that, as long as it works! Normally PHPStorm would add that automatically for me but it's possible I typed it in differently without using any type helpers.

Reply:
Cool! I can make that change and amend your commit.